torch/_dynamo/variables/dicts.py

```diff
@@ -531,6 +531,53 @@ def getitem_const(self, tx: "InstructionTranslator", arg: VariableTracker):
         raise RuntimeError("Illegal to getitem on a set")
 
 
+class FrozensetVariable(SetVariable):
+    def __init__(
+        self,
+        items: List[VariableTracker],
+        **kwargs,
+    ) -> None:
+        super().__init__(items, **kwargs)
+
+    def debug_repr(self):
+        if not self.items:
+            return "frozenset()"
+        else:
+            return "{" + ",".join(k.vt.debug_repr() for k in self.items.keys()) + "}"
+
+    @property
+    def set_items(self):
+        return frozenset(self.items.keys())
```

Comment:
Curious .. why `frozenset` creation here? Why not just

```
    @property
    def set_items(self):
        return self.items.keys()
 ```

Reply:
you're right, self.items.keys() is enough here.